app/Event.php

```diff
@@ -85,4 +85,9 @@ public function scopeBetweenTiming($query, $start, $end)
     {
         $query->whereBetween('timing', [$start, $end]);
     }
+
+    public function gallery($id)
+    {
+        return $this->hasMany('App/Image')->where('event_id',$id);
+    }
```

Comment:
you don't need the where clause. hasMany is a relation that will relate the current event (this) with any photo having  event_id = $this->id